.github/workflows/glpi.yml

```diff
@@ -110,3 +110,9 @@ jobs:
           outputs: "type=registry"
           pull: true
           tags: "${{ steps.variables.outputs.tags }}"
+      - name: "Update repository description"
+        uses: peter-evans/dockerhub-description@v4
+        with:
+            username: ${{ secrets.DOCKER_HUB_USERNAME }}
+            password: ${{ secrets.DOCKER_HUB_TOKEN }}
+            repository: glpi-project/docker-images
```

Comment:
I think a dedicated workflow with the following trigger would be more efficient.

```yaml
on:
  push:
    branches:
      - "main"
    paths:
      - "README.md"
```

Reply:
Indeed